### ISSUE TITLE
Drawing lets go a microsecond

### ISSUE THREAD

**Issue by slabzzz** [zsviczian/obsidian-excalidraw-plugin#613]
When I use my apple pencil/ipad on excalidraw, it works. But something concerning is how sometimes the drawing stops for a microsecond. This results in some gaps in my drawings.

**Comment by zsviczian** [comment]
This is due to my autosave function. I'll see what I can do to improve the situation.

**Comment by cdmoss** [comment]
Bumping this. The plugin is fantastic, this is the only issue preventing me from using it as my daily driver.

**Comment by zsviczian** [comment]
I released a fix in 1.6.32. Can you test if you are still experiencing this issue?

**Comment by zsviczian** [comment]
The freedraw tool is not designed as a hand writing tool, but as a drawing tool. The warping of text is due to the settings of the perfect freehand component in Excalidraw. It's been on my todo for quite some time to expose these settings on plugin settings, then you'll have more control over how lines are processed.

**Comment by slabzzz** [comment]
Makes sense. It's just a bit hard to explain... Using apple pencil delays the drawings, there are always gaps when I hand write, I know it's not a hand writing tool but it would've been pretty useful to do math on here. The only issue is that the text seems to get warped quite a bit, and there are gaps when hand writing. Best case scenario would've been no input delay, no text warping, and no gaps when hand writing.

**Comment by slabzzz** [comment]
Here's an overdue representation of what I'm describing:

<img width="868" alt="bug" src="https://user-images.githubusercontent.com/100169858/196019648-bdb50634-6cea-4e12-9443-c8ee0515bcdd.png">

Hard to notice but it itches the brain sometimes.

**Comment by zsviczian** [comment]
@slabzzz, version 1.7.24 should have solved the issue, but I forgot to update dependencies and as a consequence it did not... please test with 1.7.25 once it is released.

**Comment by slabzzz** [comment]
Still happens on 1.7.25. Could it be on my end?

**Comment by zsviczian** [comment]
There is not much I can do. This needs to be solved at the excalidraw package level. This is the change that was also deployed to Obsidian, and which as you report does not solve the issue: https://github.com/excalidraw/excalidraw/pull/5727
